.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: python
```

Comment:
Python is not used at all, because everything is executed in the docker container, which include Python already. This can me changed to `language:minimal`.